chia/wallet/trading/trade_store.py

```diff
@@ -236,6 +295,27 @@ async def get_trade_record_with_status(self, status: TradeStatus) -> List[TradeR
 
         return records
 
+    async def get_coins_of_interest_with_trade_statuses(self, trade_statuses: List[TradeStatus]) -> List[Coin]:
```

Comment:
I think this function should return a list of coin_ids instead. It would make the sql query simpler and it would save the caller from computing the name